### PR TITLE
update to gamma.1

### PR DESCRIPTION
**Pull request by vjjft**
Primary:
- added `nets.ts` file; replaced `capi.config.ts`
- ran `capi sync`
- updated example code to current api structure
- updated npm run scripts
- updated net config to use dev target

Ancillary:
- bumped polkadot to 0.9.40

### CODE REVIEW

**Comment by vjjft** [comment]
@harrysolovay @tjjfvi when you have a chance, if you can review, that'd be good 🙂

(It seems I can't request a review through the ui.)

**Review by harrysolovay — APPROVED** [review]

File: index.ts [hunk 1]
```diff
@@ -1,3 +1,3 @@
-import { System } from "@capi/polkadot-dev"
+import { polkadot } from "@capi/polkadot"
 
-console.log(await System.Account.entryPage(10, null).run())
+console.log( await polkadot.System.Account.entries({limit:10}).run() )
```
Comment (tjjfvi) [inline index.ts]:
```suggestion
console.log(await polkadot.System.Account.entries({ limit: 10 }).run())
```

File: nets.ts [hunk 2]
```diff
@@ -0,0 +1,16 @@
+
+import { bins, net } from "capi/nets"
+
+const bin = bins({
+  polkadot: ["polkadot-fast", "v0.9.38"],
+  polkadotParachain: ["polkadot-parachain-fast", "v0.9.380"],
```
Comment (tjjfvi) [inline nets.ts]:
```suggestion
```

Comment (vjjft) [reply nets.ts]: Is that no longer a valid bin specification?

Comment (harrysolovay) [reply nets.ts]: Ah, good catch T6. It is valid, but it's unused.

File: nets.ts
Comment (harrysolovay) [inline nets.ts:1]: Should we re-add `polkadotDev` and add a `dev` script to the `package.json` scripts?

Comment (harrysolovay) [reply nets.ts:1]: As well as specifying `polkadotDev` as a `target` of the `polkadot` net spec?

File: package.json [hunk 3]
```diff
@@ -2,17 +2,17 @@
...
   },
   "devDependencies": {
     "ts-node": "^10.9.1"
   },
   "scripts": {
     "capi": "capi",
-    "sync": "capi sync --package-json package.json",
-    "main": "capi serve -- ts-node-esm index.ts"
+    "sync": "capi sync node",
+    "main": "capi serve -- npm run run",
```
Comment (harrysolovay) [inline package.json]: This shouldn't require the leading `capi serve --`

Comment (harrysolovay) [reply package.json]: How about...

```json
// ...
"main": "node --loader ts-node/esm index.ts",
"dev": "capi serve --target dev -- npm run main"
```

File: nets.ts (hunk 2 above)
Comment (vjjft) [reply nets.ts]: *(deprecated)*

File: package.json (hunk 3 above)
Comment (vjjft) [reply package.json]: done in https://github.com/paritytech/capi-node-example/pull/3/commits/3e10c4ea602b1931720b1cd33215a60b7692b495

File: nets.ts
Comment (vjjft) [reply nets.ts:1]: done in https://github.com/paritytech/capi-node-example/pull/3/commits/3e10c4ea602b1931720b1cd33215a60b7692b495

**Comment by vjjft** [comment]
When I `npm run main` I get output, but when I `npm run dev` I get no output.

Is that expected because the local `polkadot-dev` chain has no content?

**Comment by vjjft** [comment]
When I now repeat `sync` a different url/hash is generated. (I have not diffed the `.tar` contents.)

Is that expected? If so, why?

**Comment by vjjft** [comment]
*(updated pr description)*

**Review by tjjfvi — APPROVED** [review]

**Pull request merged**